ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollView.java

```diff
@@ -21,10 +21,12 @@
 import android.util.Log;
 import android.view.MotionEvent;
 import android.view.View;
+import android.view.ViewGroup;
 import android.widget.OverScroller;
 import android.widget.ScrollView;
 
 import com.facebook.react.bridge.ReactContext;
+import com.facebook.common.logging.FLog;
```

Comment:
You'll need to add this as a dependency in the buck file... that's why the builds are failing. Or just remove this and the FLog statement below, I don't think they're that useful ;)